administrator-manual/en/accounts.rst

```diff
@@ -35,7 +35,7 @@ Samba Active Directory
 
 When installing Samba Active Directory, the system needs an additional IP address which will be the address of 
 Active Directory controller inside the LAN.
-The additional IP address must satisfy three conditions:
+The additional IP address is virtual whether or not are using you are |product| has a virtual, and must satisfy three conditions:
```

Comment:
Thank you for this PR @JeffBales !

It's not clear to me what do you mean by *virtual* here. I merge this PR anyway and fix this statement from another PR.